r/README.Rmd

```diff
@@ -36,17 +36,26 @@ Conda users on Linux and macOS can install `arrow` from conda-forge with
 conda install -c conda-forge r-arrow
 ```
 
-On macOS and Windows, installing a binary package from CRAN will handle Arrow's C++ dependencies for you. On Linux, unless you use `conda` you'll need to first install the C++ library. See the [Arrow project installation page](https://arrow.apache.org/install/) to find pre-compiled binary packages for some common Linux distributions, including Debian, Ubuntu, and CentOS. You'll need to install `libparquet-dev` on Debian and Ubuntu, or `parquet-devel` on CentOS. This will also automatically install the Arrow C++ library as a dependency. Other Linux distributions must install the C++ library from source.
+On macOS and Windows, installing a binary package from CRAN will handle
+Arrow's C++ dependencies for you. On Linux, unless you use `conda`,
+the R package will have to compile its bindings from source
+and it will need to find or download the C++ dependencies.
+As of the 0.16.0 release, this dependency resolution is automatic on most
+common Linux distributions. See `vignette("install", package = "arrow")`
+for details.
 
-If you install the `arrow` package from source and the C++ library is not found, the R package functions will notify you that Arrow is not available. Call
+If you install the `arrow` package from source and the C++ library is not found,
+the R package functions will notify you that Arrow is not available. Call
 
 ```r
 arrow::install_arrow()
 ```
 
 for version- and platform-specific guidance on installing the Arrow C++ library.
 
-When installing from source, if the R and C++ library versions do not match, installation may fail. If you've previously installed the libraries and want to upgrade the R package, you'll need to update the Arrow C++ library first.
+When installing from source, if the R and C++ library versions do not match,
+installation may fail. If you've previously installed the libraries
```

Comment:
`the`?

Reply:
Don't think it's necessary; anyway I was just adding line breaks, no wording changes.